package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@projectsophon/df-plugin-dev-server",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

```suggestion
  "version": "1.1.0",
```
Release process takes care of this

---

Acknowledge, modified